aequilibrae/paths/traffic_class.py

```diff
@@ -41,13 +41,10 @@ def __init__(self, name: str, graph: Graph, matrix: AequilibraeMatrix) -> None:
         self.mode = graph.mode
```

Comment:
We defer the preparation of the objects so we can set skimming even after creating  traffic classes (needed for the case of the QGIS plugin and overall better usability)